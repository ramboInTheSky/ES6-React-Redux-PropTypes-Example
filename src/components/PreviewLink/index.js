import styled from 'styled-components';
import { Button } from 'nhh-styles';

export default styled(Button)`
  font-size: inherit;
`;
