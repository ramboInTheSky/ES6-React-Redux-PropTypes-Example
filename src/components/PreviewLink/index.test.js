import React from 'react';
import { shallow } from 'enzyme';

import PreviewLink from './';

describe('<PreviewLink />', () => {
  it('renders correctly', () => {
    expect(shallow(<PreviewLink />)).toMatchSnapshot();
  });
});
