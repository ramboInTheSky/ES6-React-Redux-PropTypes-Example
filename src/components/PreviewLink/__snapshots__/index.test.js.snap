// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviewLink /> renders correctly 1`] = `
<WithTheme(Button)
  className="sc-fcdeBU eyvVKn"
/>
`;
